cairo/sys/src/lib.rs

```diff
@@ -85,50 +94,36 @@ macro_rules! debug_impl {
     };
 }
 
-#[repr(C)]
-pub struct cairo_t(c_void);
-debug_impl!(cairo_t);
+opaque!(cairo_t);
+opaque!(cairo_surface_t);
+opaque!(cairo_device_t);
+opaque!(cairo_pattern_t);
 
-#[repr(C)]
-pub struct cairo_surface_t(c_void);
-debug_impl!(cairo_surface_t);
-
-#[repr(C)]
-pub struct cairo_device_t(c_void);
-debug_impl!(cairo_device_t);
-
-#[repr(C)]
-pub struct cairo_pattern_t(c_void);
-
-#[cfg(any(feature = "xcb", feature = "dox"))]
-#[repr(C)]
-pub struct xcb_connection_t(c_void);
-#[cfg(any(feature = "xcb", feature = "dox"))]
-debug_impl!(xcb_connection_t);
+opaque!(
+    #[cfg(any(feature = "xcb", feature = "dox"))]
+    xcb_connection_t
```

Comment:
Ah, we forgot to add `#[cfg_attr(feature = "dox", doc(cfg(feature = "xcb")))]`...